src/containers/projects/new/step-1.svelte

```diff
@@ -52,7 +53,7 @@
       <span class="lb">This is shown on the project card to catch attention.</span>
       <span class="lb">Best to use 16:9 photos.</span>
     </span>
-    <ProjectImagePicker bind:value={$project.image_id} on:resize-image />
+    <ProjectImagePicker bind:value={$project.image_id} {uploading} on:uploading on:resize-image />
```

Comment:
Ugh, the event and the prop have the same name. If you could change one of them, I'd appreciate.